CONTRIBUTING.markdown

```diff
@@ -1,114 +1,171 @@
+
+## Github
+
+Submit your custom changes to Github using the following process:
+
+* Create a topic branch to hold your changes based on upstream/master:
+
+      git fetch upstream 
+      git checkout -b my-custom-change upstream/master
+
+* Commit logical units of work including a reference to your ticket in JIRA.  For example:
+
+      LPS-83432 Make the example in CONTRIBUTING imperative and concrete
+
+* Test your changes thoroughly! Consider the wide variety of operating systems, databases, application servers, and other related technologies Liferay supports. Make sure your changes in one environment don't break something in another environment.
+
+* Before pushing your branch to your fork on Github it’s often a good idea to rebase on the updated version of upstream/master:
+
+      git fetch upstream
+      git rebase upstream/master
+
+* Push changes in your branch to your fork: 
+
+      git push origin my-custom-change
+
+* If the change is for something in Portal Core or a module that resides in the Central Repo, submit the pull request to the main liferay-portal repo.  If the change is in an active Sub Repo such as: com-liferay-journal, submit the pull request to that specific repo.
+
+* In the LPS ticket, provide a link to your GitHub pull request and* You're done! Well, not quite ... be sure to respond to comments and questions to your pull request until it is closed.
+
+# Tooling
+
+Creating customizations and debugging code can be made easier using tooling.  Customizations to Sub Repos work the best with the tooling since they work like any custom module project.
+
+Install the following tools to aid in development:
+
+* Blade Tools - [https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/installing-blade-cli](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/installing-blade-cli)
+
+* IDE 
+  * Liferay IDE - [https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/liferay-ide](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/liferay-ide).   Use the [generate-modules-classpath](https://web.liferay.com/web/jorge.diaz/blog/-/blogs/debugging-liferay-7-0-generate-modules-classpath-in-eclipse-using-generate_modules_classpath-sh) script to import Portal Core and all of the modules located in Central.  
+
+  * IntelliJ -  Use [liferay-intellij](https://github.com/holatuwol/liferay-intellij) to help configure the environment.
+
+* Liferay Sub Repos can be used with the tooling like many other custom modules including importing them into an IDE using the built-in Gradle support.
+
+* If using Liferay IDE, modules that reside from within a Sub Repo can be deployed to the Liferay Server by dragging and dropping it to the server or use **'gradle deploy'** from the CLI for IntelliJ or text editors.
+
+# Additional Resources
+
 * [Liferay and JIRA](http://www.liferay.com/community/wiki/-/wiki/Main/JIRA)
```

Comment:
I think we should remove this link for the following reasons:
1) It's linking to the (very) old wiki. We should not redirect users there since most information is outdated.
2) This specific article was written a looong time ago and it's probably very out of date